src/js/elements/mapchip/mapchip.js

```diff
@@ -0,0 +1,182 @@
+import {FilterController} from "../subindicator_filter/filter_controller";
+import {Legend} from "./legend";
+import {DataFilterModel} from "../../models/data_filter_model";
+import {Observable} from "../../utils";
+
+const filterContentClass = '.map-options__filters_content';
+const mapChipBlockClass = '.map-options';
+const filterRowClass = '.map-options__filter-row';
+const legendContainerClass = '.map-options__legend_wrap';
+
+/**
+ * Represent the map chip at the bottom of the map
+ */
+export class MapChip extends Observable {
```

Comment:
You need to split this class up to use a model - it should contain:

- title
- description
- visibility (i.e. visible or hidden)
and all the appropriate events